src/test/java/org/opensearch/neuralsearch/query/HybridQueryAggregationsIT.java

```diff
@@ -182,6 +182,57 @@ public void testAggregationNotSupportedConcurrentSearch_whenUseSamplerAgg_thenSu
         }
     }
 
+    @SneakyThrows
+    public void testPostFilter_WhenConcurrentSearchNotEnabled_thenSuccessful() {
```

Comment:
can we add one more test case for case when concurrent search is enabled. It's GA since 2.12, probably will be enabled by default in coming versions